plugins/org.eclipse.n4js.utils/src/org/eclipse/n4js/utils/URIUtils.java

```diff
@@ -272,4 +263,35 @@ public static URI addEmptyAuthority(URI uri) {
 		return uri;
 	}
 
+	/** Converts the given {@link org.eclipse.emf.common.util.URI} to a {@link java.net.URI} */
+	static public java.net.URI toURI(URI uri) throws URISyntaxException {
+		return new java.net.URI(uri.toString());
+	}
+
+	/**
+	 * Converts the given URI to a {@link File} iff it is a {@link org.eclipse.emf.common.util.URI#isFile() file URI};
+	 * otherwise returns <code>null</code>.
+	 * <p>
+	 * Same as {@link org.eclipse.emf.common.util.URI#toFileString()}, but returns a {@link File} instance instead of a
+	 * string.
+	 */
+	static public File toFile(URI uri) {
+		return uri != null && uri.isFile() ? new File(uri.toFileString()) : null;
+	}
+
+	/**
+	 * Converts the given URI to a {@link Path} iff it is a {@link org.eclipse.emf.common.util.URI#isFile() file URI};
+	 * otherwise returns <code>null</code>.
+	 * <p>
+	 * Same as {@link org.eclipse.emf.common.util.URI#toFileString()}, but returns a {@link Path} instance instead of a
+	 * string.
+	 * <p>
+	 * <b>Attention: This fails on platform Windows:</b><br>
+	 * {@code Paths.get(org.eclipse.emf.common.util.URI#toFileString());} <br>
```

Comment:
Instead of the `<br>`s you could use tags `<code> ... </code>` instead of `{@code ...}`.

Reply:
You mean that `<code>` tags would implicitly add a line break? That didn't work for me.